docs/guide/migration.rst

```diff
@@ -9,4 +9,193 @@ This is a guide to migrate from Stable-Baselines to Stable-Baselines3.
+
+.. warning::
+
+	PyTorch implementation of RMSprop `differs from Tensorflow's <https://github.com/pytorch/pytorch/issues/23796>`_,
+	which leads to `different and potentially more unstable results <https://github.com/DLR-RM/stable-baselines3/pull/110#issuecomment-663255241>`_.
+	Use ``stable_baselines3.common.sb2_compat.rmsprop_tf_like.RMSpropTFLike`` optimizer to match the results
+	with Tensorflow's implementation. This can be done through ``policy_kwargs``: ``A2C(policy_kwargs=dict(optimizer_class=RMSpropTFLike))``
+
+
+PPO
+^^^
+
+- ``cliprange`` -> ``clip_range``
+- ``cliprange_vf`` -> ``clip_range_vf``
+- ``nminibatches`` -> ``batch_size``
+
+.. warning::
+
+	``nminibatches`` gave different batch size depending on the number of environments:  ``batch_size = (n_steps * n_envs) // nminibatches``
+
+
+- ``clip_range_vf`` behavior for PPO is slightly different: Set it to ``None`` (default) to deactivate clipping (in SB2, you had to pass ``-1``, ``None`` meant to use ``clip_range`` for the clipping)
+- ``lam`` -> ``gae_lambda``
+- ``noptepochs`` -> ``n_epochs``
+
+PPO default hyperparameters are the one tuned for continuous control environment.
+We recommend taking a look at the :ref:`RL Zoo <rl_zoo>` for hyperparameters tuned for Atari games.
+
+
+DQN
+^^^
+
+Only the vanilla DQN is implemented right now but extensions will follow.
+Default hyperparameters are taken from the nature paper, except for the optimizer and learning rate that were taken from Stable Baselines defaults.
+
+DDPG
+^^^^
+
+DDPG now follows the same interface as SAC/TD3.
+For state/reward normalization, you should use ``VecNormalize`` as for all other algorithms.
+
+SAC/TD3
+^^^^^^^
+
+SAC/TD3 now accept any number of critics, e.g. ``policy_kwargs=dict(n_critics=3)``, instead of only two before.
+
+
+.. note::
+
+	SAC/TD3 default hyperparameters (including network architecture) now match the ones from the original papers.
+	DDPG is using TD3 defaults.
+
+
+SAC
+^^^
+
+SAC implementation matches the latest version of the original implementation: it uses two Q function networks and two target Q function networks
+instead of two Q function networks and one Value function network (SB2 implementation, first version of the original implementation).
+Despite this change, no change in performance should be expected.
+
+.. note::
+
+	SAC ``predict()`` method has now ``deterministic=False`` by default for consistency.
+	To match SB2 behavior, you need to explicitly pass ``deterministic=True``
+
+
+
+New logger API
+--------------
+
+- Methods were renamed in the logger:
+
+  - ``logkv`` -> ``record``, ``writekvs`` -> ``write``, ``writeseq`` ->  ``write_sequence``,
+  - ``logkvs`` -> ``record_dict``, ``dumpkvs`` -> ``dump``,
+  - ``getkvs`` -> ``get_log_dict``, ``logkv_mean`` -> ``record_mean``,
+
+
+Internal Changes
+----------------
+
+Please read the :ref:`Developer Guide <developer>` section.
+
+
+New Features
```

Comment:
Could say "New Features (v1.0 versus stable-baselines)" to highlight that these are major changes compared to stable-baselines2. I do not think we will be updating this list later on.